src/utils/queue_builder.rs

```diff
@@ -92,6 +98,7 @@ pub fn queue_builder(
         }));
 
         if let Err(e) = res {
+            // TODO: how to report this?
             error!("GRAVE ERROR Building new crates panicked: {:?}", e);
```

Comment:
`e`  is `Box<dyn Any + Send>` because `res` is `std::thread::Result`. 
`error!` would at least end up in sentry. 

I don't fully understand the queue logic here, but does the comment (_trying to handle the error won't help_) mean that we should `panic::resume_unwind(e)` here?

Reply:
That comment is about locking the queue (specifically the `expect()`). It's not related to building the package; and indeed I would prefer *not* to panic so the queue will start building again once we unlock it.